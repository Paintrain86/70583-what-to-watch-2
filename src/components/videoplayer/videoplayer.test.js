import React from 'react';
import renderer from 'react-test-renderer';
import Videoplayer from './videoplayer.jsx';

describe(`Videoplayer`, () => {
  it(`component is rendered correctly`, () => {
    const testProps = {
      id: 1,
      title: `Тестовая киношка`,
      poster: `images/image.jpg`,
      previews: [
        {
          src: `movies/movie.mp4`,
          type: `video/mp4`
        }
      ],
      isNeedPlaying: false
    };

    const markup = renderer
      .create(<Videoplayer {...testProps}/>)
      .toJSON();

    expect(markup).toMatchSnapshot();
  });
});
