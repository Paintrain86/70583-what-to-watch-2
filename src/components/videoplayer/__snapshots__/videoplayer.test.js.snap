// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Videoplayer component is rendered correctly 1`] = `
<video
  poster="images/image.jpg"
  width="100%"
>
  <source
    src="movies/movie.mp4"
    type="video/mp4"
  />
  Sorry, your browser is a piece of shit
</video>
`;
